src/app/home/DetailViewComponent.tsx

```diff
@@ -47,29 +47,38 @@ interface IProps {
   watchStorageAddEditStatus: (string) => void;
 }
 
+enum DataListItems {
+  clusterList,
+  storageList,
+  planList,
+}
+
 interface IState {
   expanded: any[];
   plansDisabled: boolean;
-  isOpen: boolean;
-  isWizardOpen: boolean;
+  expand: boolean[];
   modalType: string;
 }
 
 class DetailViewComponent extends Component<IProps, IState> {
-  state = {
-    expanded: [],
-    plansDisabled: true,
-    isOpen: false,
-    isWizardOpen: false,
-    modalType: '',
-  };
+  constructor (props) {
```

Comment:
Since we are using es7 class props, we dont need the constructor or this.state. https://borgs.cybrilla.com/tils/es7-class-properties/